crates/bevy_pbr/src/deferred/pbr_deferred_functions.wgsl

```diff
@@ -28,16 +28,19 @@ fn deferred_gbuffer_from_pbr_input(in: PbrInput) -> vec4<u32> {
      // Real time occlusion is applied in the deferred lighting pass.
      // Deriving luminance via Rec. 709. coefficients
      // https://en.wikipedia.org/wiki/Rec._709
-    let diffuse_occlusion = dot(in.diffuse_occlusion, vec3<f32>(0.2126, 0.7152, 0.0722));
+     let rec_709_coeffs = vec3<f32>(0.2126, 0.7152, 0.0722);
```

Comment:
Not sure why but this line and the comment above has a wrong indentation.

Reply:
Fixed.